source/MainWindow.cpp

```diff
@@ -1,8 +1,13 @@
 #include "MainWindow.h"
 #include <Application.h>
+#include <Catalog.h>
+
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "MainWindow"
+
 //-------------------------------------------------------------------
 MainWindow::MainWindow()
-	: BWindow(BRect(240, 30, 440, 130), "The Awesome Resizer",
+	: BWindow(BRect(240, 30, 440, 130), B_TRANSLATE("The Awesome Resizer"),
```

Comment:
In case someone wants to translate the app name, B_TRANSLATE_SYSTEM_NAME would be better:

```suggestion
	: BWindow(BRect(240, 30, 440, 130), B_TRANSLATE_SYSTEM_NAME("The Awesome Resizer"),
```